department-of-veterans-affairs/vets-website
# 98143 Sign-in widget: Update the sign-in alert content

## Are you removing, renaming or moving a folder in this PR?

### :warning: TeamSites :warning:
Examples of a TeamSite: https://va.gov/health and https://benefits.va.gov/benefits/. This scenario is also referred to as the "injected" header and footer. You can reach out in the `#sitewide-public-websites` Slack channel for questions.

Did you change site-wide styles, platform utilities or other infrastructure?

## Summary

Change the sign-in alert that appears on unauthenticated/Drupal pages needs to reflect that Login.gov and IDme are the two supported login providers, and that identity verification is required

## Related issue(s)

- department-of-veterans-affairs/va.gov-team/issues/98143
- department-of-veterans-affairs/va.gov-team/issues/97279

## Testing done

- Manual testing in local development:
  - 'yarn watch` from `vets-website`
  - `foreman start -m all=1,clamd=0,freshclam=0` from `vets-api`
  - Visit `http://localhost:3001/health-care/schedule-view-va-appointments/` to check that alert content is updated

## Screenshots

|         | Before | After |
| ------- | ------ | ----- |
| Mobile  |    <img width="385" alt="Screenshot 2024-12-12 at 10 50 15 AM" src="https://github.com/user-attachments/assets/d1862e99-e950-461f-999d-ec0f42ff3daa" />    |   <img width="387" alt="Screenshot 2024-12-12 at 10 52 53 AM" src="https://github.com/user-attachments/assets/67d1b53d-2978-480b-9ee3-78f089ac886d" />    |
| Desktop |   <img width="684" alt="Screenshot 2024-12-12 at 10 51 10 AM" src="https://github.com/user-attachments/assets/4af1c250-d82a-4329-817c-727744352010" />     |  <img width="684" alt="Screenshot 2024-12-12 at 10 51 31 AM" src="https://github.com/user-attachments/assets/4760e085-9f87-43b9-8888-c33bcd2501e3" />     |

## What areas of the site does it impact?

These changes impact _widgets_ that get embedded on static pages. The following are some paths that embed widgets that used this alert.

- `/health-care/schedule-view-va-appointments/`
- `/health-care/refill-track-prescriptions/`
- `/health-care/secure-messaging/`
- `/health-care/view-test-and-lab-results/`
- `health-care/get-medical-records/`

## Acceptance criteria

### Quality Assurance & Testing


### Error Handling


### Authentication


## Requested Feedback

Question: Should I update  ServiceProvidersText.jsx for this PR? I decided not to.
I decided to only update the SignIn alert for `src/applications/static-pages/cta-widget` to avoid breaking the other apps that are presently using `ServiceProvidersText`.

❯ rg ServiceProvidersText -l src 
src/platform/user/authentication/components/ServiceProvidersText.jsx
src/platform/user/package.json
src/platform/user/tests/authentication/components/ServiceProvidersText.unit.spec.js
src/platform/user/exportsFile.js
src/applications/vre/28-1900/containers/ConfirmationPage.jsx
src/applications/financial-status-report/components/shared/UnverifiedPrefillAlert.jsx
src/applications/vre/25-8832/containers/ConfirmationPage.jsx
src/applications/static-pages/download-1095b/components/App/index.js
src/applications/static-pages/medical-copays-cta/components/App/index.js
src/applications/disability-benefits/686c-674-v2/containers/ConfirmationPage.jsx
